File: Yang2018/model/name2idx/variables.jl
module V
const var_names = [
    "p53"
    "Mdm2"
    "Wip1"
    "ATMp"
];

for (idx,name) in enumerate(var_names)
    eval(Meta.parse("const $name = $idx"));
end

const len_f_vars = length(var_names);

end  # module

File: Heldt2018/model/name2idx/variables.py
var_names = [
    'uRb',
    'tE2f',
    'E2f',
    'tE1',
    'tP21',
    'tCe',
    'tCa',
    'CeP21',
    'CaP21',
    'C1',
    'E1C1',
    'aPcna',
    'iPcna',
    'Rc',
    'pRc',
    'aRc',
    'iRc',
    'Dna',
    'P53',
    'Dam',
    'Pr',
]

for idx, name in enumerate(var_names):
    exec(
        '{} = {:d}'.format(
            name, idx
        )
    )

len_f_vars = len(var_names)